test/distributed/_composable/fsdp/test_fully_shard_logging.py

```diff
@@ -0,0 +1,61 @@
+# Owner(s): ["module: fsdp"]
+import functools
+import os
+import unittest.mock
+
+import torch.distributed as dist
+from torch._dynamo.test_case import run_tests
+from torch.testing._internal.common_distributed import skip_if_lt_x_gpu
+from torch.testing._internal.inductor_utils import HAS_CUDA
+from torch.testing._internal.logging_utils import LoggingTestCase
+
+requires_cuda = unittest.skipUnless(HAS_CUDA, "requires cuda")
+requires_distributed = functools.partial(
+    unittest.skipIf, not dist.is_available(), "requires distributed"
+)
+
+
+@skip_if_lt_x_gpu(2)
+class LoggingTests(LoggingTestCase):
+    @requires_distributed()
+    def test_fsdp_logging(self):
+        env = dict(os.environ)
+        env["TORCH_LOGS"] = "fsdp"
+        env["RANK"] = "0"
+        env["WORLD_SIZE"] = "1"
+        env["MASTER_PORT"] = "34715"
+        env["MASTER_ADDR"] = "localhost"
+        stdout, stderr = self.run_process_no_exception(
+            """\
+import logging
+import torch
+import torch.distributed as dist
+import torch.nn as nn
+from torch.distributed._composable.fsdp import fully_shard
+logger = logging.getLogger("torch.distributed._composable.fsdp")
+logger.setLevel(logging.DEBUG)
```

Comment:
What is the default log level? Do we have to set it to `DEBUG` here, or is it just to emphasize that we need it at `DEBUG`?

Reply:
The default lever for the TORCH_LOGS should be `WARN`, but we run with `TORCH_LOGS=+fsdp` to print in `'DEBUG'` level.
However in the test here, the output will be decided by the default level here, so only setting of `DEBUG `could have the logging results in `stderr`.